libs/blockprotocol/cli/commands/publish.js

```diff
@@ -183,7 +183,7 @@ const run = async (providedOptions) => {
   if (errors || !block) {
     const errorMsg = errors?.[0]?.msg;
     console.log(chalk.red(errorMsg));
-    process.exit();
+    process.exit(1);
```

Comment:
This particular change should fail an error [like this](https://github.com/hashintel/hash/actions/runs/4281357191/jobs/7454344294#step:5:12) next time (which is what we want):

```
––––––––––––––––––––––––––––––––––––––––––––––––
block-metadata.json must contain a 'blockType' property
```